ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -775,16 +878,12 @@ export const RuntimePanel = fp.flow(
            aria-label='Delete Environment'
            disabled={![RuntimeStatus.Running, RuntimeStatus.Stopped].includes(status as RuntimeStatus)}
            onClick={() => setPanelContent(PanelContent.Delete)}>Delete Environment</Link>
-         <UpdateRuntimeButton
-           disabled={shouldDisableUpdateButton(runtimeChanged, runtimeExists, status)}
-           label={runtimeExists ? 'Update' : 'Create'}
-           onUpdate={() => onUpdate()}
-           selectedDiskSize={selectedDiskSize}
-           selectedMachineType={selectedMachineType}
-           selectedDataprocConfig={selectedDataprocConfig}
-           setRequestedRuntime={(value) => setRequestedRuntime(value)}
-         />
+         {!runtimeExists ? renderCreateButton() :
+           runtimeDiffs.map(diff => diff.differenceType).includes(RuntimeDiffState.NEEDS_DELETE) ? renderNextButton() :
```

Comment:
Huh - wait. This actually isn't clear in the mocks, but I thought that the confirmation page would appear in all cases (whether this was an update OR a delete/recreate). I don't feel strongly on this though, did you discuss this point with Lou / Shimon already?